packages/api/bin/cli.js

```diff
@@ -77,5 +78,21 @@ program
     }
   });
 
+program
```

Comment:
Should this get added to the README along with the reindexing commands? CLI `help` is fine too, but may be good to document this in the README as well